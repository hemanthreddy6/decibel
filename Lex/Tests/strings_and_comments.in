/* This is a comment **** **/ This_is_not_a_comment   /* Another * comment */ 3.1415926535 // yet another one // still comment */ /* fdsfj */
'This is a string. "This is a string inside a string"' id 'another string'
" This string's got a comment in it. // but it's not a comment /* and this isn't a comment either */" another_id "another string" ""
"
multiline string?
"
'multiline strings also work,
// and newline characters are part of that string.' 123456
// "this is a string inside a comment so still a comment"
'"huh?"
// a comment at the end of the file!
