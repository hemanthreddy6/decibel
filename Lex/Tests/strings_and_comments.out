IDENTIFIER
FLOAT
STRING
IDENTIFIER
STRING
STRING
IDENTIFIER
STRING
STRING
STRING
STRING
NUMBER
INVALID_SYMBOL
STRING
